<launch>

    <!-- bag1 -->
    <param name="initial_x" value="0.00815962441266" />
    <param name="initial_y" value="0.003059721319" />
    <param name="initial_t" value="0.0" />   
    <param name="initial_secs" value="1649348542" />
    <param name="initial_nsecs" value="208360780" />
    <node pkg="tf2_ros" type="static_transform_publisher" name="odom" output="screen" args="0.00815962441266 0.003059721319 0 0 0 0 1 world odom" />

    <!-- bag2 -->
    <!-- <param name="initial_x" value="0.0242218170315" /> -->
    <!-- <param name="initial_y" value="0.00374252465554" /> -->
    <!-- <param name="initial_t" value="0.0" /> -->
    <!-- <param name="initial_secs" value="1649348648" /> -->
    <!-- <param name="initial_nsecs" value="859687942" /> -->
    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="odom" output="screen" args="0.0242218170315 0.00374252465554 0 0 0 0 1 world odom" /> -->

    <!-- bag3 -->
    <!-- <param name="initial_x" value="0.00546012399718" /> -->
    <!-- <param name="initial_y" value="0.0399703718722" /> -->
    <!-- <param name="isnitial_t" value="0.0" /> -->
    <!-- <param name="initial_secs" value="1649348784" /> -->
    <!-- <param name="initial_nsecs" value="993775398" /> -->
    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="odom" output="screen" args="0.00546012399718 0.0399703718722 0 0 0 0 1 world odom" /> -->
 
    <node pkg="project1" type="velocities" name="velocities_calculator" output="screen"/>
    <node pkg="project1" type="odometry" name="odometry_calculator" output="screen"/>
    <node pkg="project1" type="control" name="control_calculator" output="screen"/>
    <node pkg="project1" type="calibration" name="calibration_calculator" output="screen"/>

</launch>
